src/Api/Models/Response/ConfigResponseModel.cs

```diff
@@ -46,6 +47,7 @@ public class ServerConfigResponseModel
 
 public class EnvironmentConfigResponseModel
 {
+    public string CloudVaultRegion { get; set; }
```

Comment:
Just saw the discussion in Slack re: naming. I agree with @r-tome and @shane-melton that this doesn't need to refer to "Vault", especially because it's just sending a region and not a URL. It's only used to set the cloud vault URL currently, but it might be reused in the future, so a more general name could be a benefit.

I suggest `CloudRegion`.